packages/gestalt/src/ScrollableContainer.css

```diff
@@ -0,0 +1,3 @@
+.customClass {
```

Comment:
What is this class used for? Is it being used?

Reply:
REMOVED